tests/framework/ROM/TimeSeries/SyntheticHistory/auto_arma.xml

```
@@ -0,0 +1,159 @@
+      <Input class="Models" type="ROM">synth</Input>
+      <Output class="DataObjects" type="DataSet">romMeta</Output>
+      <Output class="OutStreams" type="Print">romMeta</Output>
+    </IOStep>
+    <MultiRun name="sample">
+      <Input class="DataObjects" type="PointSet">placeholder</Input>
+      <Model class="Models" type="ROM">synth</Model>
+      <Sampler class="Samplers" type="MonteCarlo">mc</Sampler>
+      <Output class="DataObjects" type="HistorySet">samples</Output>
+      <Output class="OutStreams" type="Print">samples</Output>
+    </MultiRun>
+    <IOStep name="plot">
+      <Input class="DataObjects" type="HistorySet">indata</Input>
+      <Input class="DataObjects" type="HistorySet">samples</Input>
+      <Output class="OutStreams" type="Plot">compare_0</Output>
+      <Output class="OutStreams" type="Plot">compare_1</Output>
+    </IOStep>
+  </Steps>
+
+  <Files>
+    <Input name="infile">../TrainingData/GlobalFourierARMA_A.csv</Input>
+  </Files>
+
+  <Samplers>
+    <MonteCarlo name="mc">
+      <samplerInit>
+        <limit>2</limit>
+        <initialSeed>42</initialSeed>
+      </samplerInit>
+      <constant name="scaling">1.0</constant>
+    </MonteCarlo>
+  </Samplers>
+
+  <Models>
+    <ROM name="synth" subType="SyntheticHistory">
+      <Target>signal0, signal1, pivot</Target>
+      <Features>scaling</Features>
+      <pivotParameter>pivot</pivotParameter>
+      <fourier target="signal0,signal1">
+        <periods>2, 5, 10, 20, 50</periods>
+      </fourier>
+      <gaussianize target="signal0, signal1"/>
+      <autoarma target="signal0, signal1" seed='42'>
+        <P_upper>5</P_upper>
+        <Q_upper>5</Q_upper>
+        <criterion>bic</criterion>
+      </autoarma>
+      <arma target="signal0, signal1" auto_select="True" seed='42'>
+        <P>1</P> <!-- NOTE: because `auto-select` is True, these are ignored -->
+        <Q>0</Q> <!-- NOTE: because `auto-select` is True, these are ignored -->
+      </arma>
```

I suggest instead of ignoring the user-provided P and Q, if auto_select = "True", either to error out or to consider them as P_upper, or lower, and the same for Q.

---

P_upper and Q_upper are already used in the `autoarma` algorithm, so I think maybe I'll just have it error out if the user provides P and Q here when `auto-select` is True